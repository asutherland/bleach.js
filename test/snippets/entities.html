<p>Something &amp; nothing, &lt;script src="bad.js"&gt;&lt;&#47;script&#62;</p>