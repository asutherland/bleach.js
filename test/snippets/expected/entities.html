Something & nothing, <script src="bad.js"></script>